### PR TITLE
Mgmt: Cosmos update to 2020 09

### PR DESCRIPTION
**Pull request by ChenTanyi**

### CODE REVIEW

File: sdk/resourcemanager/azure-resourcemanager-cosmos/src/main/java/com/azure/resourcemanager/cosmos/implementation/CosmosDBAccountImpl.java [hunk 1]
```diff
@@ -362,7 +368,17 @@ public CosmosDBAccountImpl withDataModelGremlin() {
 
     @Override
     public CosmosDBAccountImpl withIpRangeFilter(String ipRangeFilter) {
-        this.innerModel().withIpRangeFilter(ipRangeFilter);
+        List<IpAddressOrRange> rules = new ArrayList<>();
+        for (String ip : ipRangeFilter.split(",")) {
+            rules.add(new IpAddressOrRange().withIpAddressOrRange(ip));
+        }
```
Comment (weidongxu-microsoft) [inline sdk/resourcemanager/azure-resourcemanager-cosmos/src/main/java/com/azure/resourcemanager/cosmos/implementation/CosmosDBAccountImpl.java:374]: `CoreUtils.isNullOrEmpty`

**Review by weidongxu-microsoft — APPROVED** [review]

File: sdk/resourcemanager/azure-resourcemanager-cosmos/src/main/java/com/azure/resourcemanager/cosmos/implementation/CosmosDBAccountImpl.java (hunk 1 above)
Comment (ChenTanyi) [reply sdk/resourcemanager/azure-resourcemanager-cosmos/src/main/java/com/azure/resourcemanager/cosmos/implementation/CosmosDBAccountImpl.java]: done

File: sdk/resourcemanager/azure-resourcemanager-cosmos/src/main/java/com/azure/resourcemanager/cosmos/implementation/CosmosDBAccountImpl.java [hunk 2]
```diff
@@ -76,7 +77,12 @@ public DatabaseAccountOfferType databaseAccountOfferType() {
 
     @Override
     public String ipRangeFilter() {
-        return this.innerModel().ipRangeFilter();
+        return this.ipRules().stream().map(IpAddressOrRange::ipAddressOrRange).collect(Collectors.joining(","));
+    }
+
+    @Override
+    public List<IpAddressOrRange> ipRules() {
+        return this.innerModel().ipRules();
```
Comment (xccc-msft) [inline sdk/resourcemanager/azure-resourcemanager-cosmos/src/main/java/com/azure/resourcemanager/cosmos/implementation/CosmosDBAccountImpl.java:89]: Collections.unmodifiableList?

Comment (ChenTanyi) [reply sdk/resourcemanager/azure-resourcemanager-cosmos/src/main/java/com/azure/resourcemanager/cosmos/implementation/CosmosDBAccountImpl.java]: done

**Review by xccc-msft — APPROVED** [review]

**Pull request merged**